### DISCUSSION TITLE
Cómo adaptar el salto de palabra

### DISCUSSION THREAD

**Discussion by iciaparicio** [Telefonica/mistica-design#925] [Proposals]
Aquí la conversación completa de Teams:
https://teams.microsoft.com/l/message/19:29466ca25d9b4b5e82f023545edae231@thread.skype/1665128057069?tenantId=9744600e-3e04-492e-baa1-25ec245c6f10&groupId=bef9be07-03e4-4afb-8a79-f720a63e4c28&parentMessageId=1665128057069&teamName=Novum&channelName=DesignCore-Public&createdTime=1665128057069&allowXTenantAccess=false

**Comment by yceballost** [comment]
Esto estaba resuelto, right?

**Reply by aweell** [reply]
Está resuelto en https://github.com/Telefonica/mistica-design/issues/932
